packages/tpetra/core/ext/TpetraExt_MatrixMatrix_def.hpp

```diff
@@ -890,42 +809,43 @@ add (const Scalar& alpha,
   MM = Teuchos::null;
   MM = rcp(new TimeMonitor(*TimeMonitor::getNewTimer(prefix_mmm + std::string("Tpetra::Crs constructor"))));
 #endif
-  //      C = rcp(new crs_matrix_type(CrowMap, CcolMap, rowptrs, colinds, vals, params));
   C.replaceColMap(CcolMap);
```

Comment:
It's a bit goofy that we create an empty C, then call a bunch of methods on it. CrsMatrix has a constructor that takes everything a fill-complete matrix needs, including all four Maps, the Import and Export, and all the local data.

Reply:
My comment still stands, but we can fix that later.